superset/views/core.py

```diff
@@ -2731,6 +2731,17 @@ def sqllab(self) -> FlaskResponse:
             "superset/basic.html", entry="sqllab", bootstrap_data=bootstrap_data
         )
 
+    @has_access
+    @expose("/sqllab/search", methods=["GET"])
+    def sqllab_search(self) -> FlaskResponse:
+        if not (
+            is_feature_enabled("ENABLE_REACT_CRUD_VIEWS")
+            and is_feature_enabled("SIP_34_QUERY_SEARCH_UI")
+        ):
+            return redirect("/superset/sqllab#search", code=307)
+
+        return super().render_app_template()
```

Comment:
instead of this being a `super` function, maybe it should just return a `render_crud_views_template` that lives in the crud directory?

Reply:
Sure we could make this a function instead of a method on the base class. I don't see any big advantages there. We've been using this pattern for all other FAB CRUD view replacements, eg https://github.com/preset-io/incubator-superset/blob/091432ea8ea4ddd7281eaf41c261f006fe99b352/superset/views/database/views.py#L104

Reply:
If this is of concern we can open an issue to clean up how the app templates are rendered for these CRUD view replacements. There are a few other follow up PRs that are waiting for this to be merged so I'm hesitant in adding "refactor all routing/template logic for crud views" to the current scope.